client/lib/abtest/active-tests.js

```diff
@@ -45,6 +45,14 @@ module.exports = {
 		},
 		defaultVariation: 'original'
 	},
+	sidebarPlanLink: {
+		datestamp: '20160101',
+		variations: {
+			plans: 50,
+			'plans/my-plan': 50,
```

Comment:
I disagree. Lack of common naming scheme between tests, nudges, component names, funnels and event names bothers me.

In my opinion, the following should be similar if possible:

- test name
- nudge name
- componoent name
- url
- cta_name